app/uk/gov/hmrc/hmrcemailrenderer/templates/customs/customsRegistrationSuccessful.scala.html

```diff
@@ -2,14 +2,29 @@
 @uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params,
     "Register for customs services - application successful", showHeading1 = false) {
 
-@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.salutation(params)
+<p style="margin: 0 0 30px; font-size: 19px;">@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.salutationFullName(params)</p>
 
-<p style="margin: 0 0 30px; font-size: 19px;">You have successfully registered for customs services.</p>
+<p style="margin: 0 0 30px; font-size: 19px;">@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.organisationname.html.orgName(params) now registered for Customs Declaration Service (CDS).</p>
 
-<p style="margin: 0 0 30px; font-size: 19px;">Your EORI number is now active and in your customs account. When trading you should only share this number with trusted business partners.</p>
+<p style="margin: 0 0 30px; font-size: 19px;">Active from : @{params.getOrElse("completionDate", "")}</p>
```

Comment:
If there isn't a completionDate, then this will just display an empty string. It might be worth putting something more meaningful, just in case.